HOWTOCONTRIBUTE.rst

```diff
@@ -0,0 +1,231 @@
+=====================
+Contribution Guide
+=====================
+
+Contributions are highly welcomed and appreciated.  Every little help counts,
+so do not hesitate! You can make a high impact on ``xskillscore`` just by using it and
+reporting `issues <https://github.com/raybellwaves/xskillscore/issues>`__.
+
+The following sections cover some general guidelines
+regarding development in ``xskillscore`` for maintainers and contributors.
+
+
+Nothing here is set in stone and can't be changed.
+Feel free to suggest improvements or changes in the workflow.
+
+
+
+.. contents:: Contribution links
+   :depth: 2
+
+
+
+.. _submitfeedback:
+
+Feature requests and feedback
+-----------------------------
+
+We are eager to hear about your requests for new features and any suggestions about the
+API, infrastructure, and so on. Feel free to submit these as
+`issues <https://github.com/raybellwaves/xskillscore/issues/new>`__ with the label "feature request."
+
+Please make sure to explain in detail how the feature should work and keep the scope as
+narrow as possible. This will make it easier to implement in small PRs.
+
+
+.. _reportbugs:
+
+Report bugs
+-----------
+
+Report bugs for ``xskillscore`` in the `issue tracker <https://github.com/raybellwaves/xskillscore/issues>`_
+with the label "bug".
+
+If you are reporting a bug, please include:
+
+* Your operating system name and version.
```

Comment:
Given that you added the version tool you could add something like `xs.show_versions()` or whatever the command is

Reply:
I still get 
```
>>> xs.__version__
'0.0.post249+dirty'`